client/src/main/scala/io/delta/sharing/client/DeltaSharingFileSystem.scala

```diff
@@ -154,6 +163,7 @@ private[sharing] class DeltaSharingFileSystem extends FileSystem {
     throw new UnsupportedOperationException("mkdirs")
 
   override def getFileStatus(f: Path): FileStatus = {
+    logInfo(s"Checking delta sharing file status for path: $f.")
```

Comment:
Question: Is it possible to guard these with some debug level?

Reply:
updated to logDebug